src/votekit/profile.py

```diff
@@ -55,14 +55,14 @@ def num_ballots(self) -> Fraction:
 
     def to_dict(self) -> dict:
         """
-        Converts balots to dictionary with keys, ranking and
+        Converts balots to dictionary with keys, ranking, tuple and
```

Comment:
small typo "bal[l]ots" 

also this docstring could be more clear, I had to read it a few times while looking at the code to understand what it was saying.

Maybe something like "Converts ballots to a dict mapping rankings to weights"